homeassistant/components/hive/sensor.py

```diff
@@ -48,6 +48,28 @@
         device_class=SensorDeviceClass.TEMPERATURE,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    SensorEntityDescription(
+        key="Heating_Current_Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        icon="mdi:thermometer",
+    ),
+    SensorEntityDescription(
+        key="Heating_Target_Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        icon="mdi:thermometer",
+    ),
+    SensorEntityDescription(
+        key="Heating_Mode",
+        icon="mdi:radiator",
+    ),
+    SensorEntityDescription(
+        key="Hotwater_Mode",
+        icon="mdi:hand-water",
+    ),
```

Comment:
What were the possible states for these?

Reply:
ON, OFF, SCHEDULE

Reply:
Can we make them lowercase and make these an enum sensor so you know what the values can be beforehand?

Reply:
Converted these to enums

Reply:
Temporarily removed the changes to this file, so that the rest of the PR can be progressed.